tests/test_apply.py

```diff
@@ -1262,6 +1262,33 @@ def circuit():
         assert np.allclose(res_sv, expected_sv, atol=tol, rtol=0)
         assert np.allclose(res_probs, expected_prob, atol=tol, rtol=0)
 
+    @pytest.mark.parametrize(
```

Comment:
Is there any plan to test it against `LGPU-MPI`?

Reply:
yeah, I'll run it after applying the first-round of suggestions. Thanks @multiphaseCFD